src/index.ts

```diff
@@ -65,56 +65,52 @@ export async function createUnifont<T extends Provider[]>(providers: T, unifontO
 
   const allProviders = Object.keys(stack)
 
-  async function resolveFont(fontFamily: string, options?: Partial<ResolveFontOptions>, providers = allProviders) {
-    const mergedOptions = { ...defaultResolveOptions, ...options }
-    for (const id of providers) {
-      const provider = stack[id]
+  return {
+    async resolveFont(fontFamily, options, providers = allProviders) {
```

Comment:
is there a reason you're defining them inline within the object?

it feels like there's no particular reason to switch, and I preferred keeping them out from a stylistic point of view. (it also makes refactoring easier.)